deepchecks/base/dataset.py

```diff
@@ -184,9 +192,9 @@ def from_numpy(
             *args: (np.ndarray):
                 expecting it to contain two numpy arrays (or at least one),
                 first with features, second with labels.
-            feature_names (Sequence[Hashable], default None):
-                names for the feature columns. If not provided next names will
-                be assigned to the feature columns: X1-Xn (where n - number of features)
+            columns (Sequence[Hashable], default None):
+                names for the columns. If not provided next names will
+                be assigned to the feature columns: 1-n (where n - number of features)
```

Comment:
```suggestion
                names for the columns. If none provided, the names that will be automatically assigned
                to columns will be: 1-n (where n - number of columns)
```

Reply:
in addition, why 1-n? I think that X_1 to X_n (or f_1 to f_n) is better. traditionally avoids confusion between column names and index names

Reply:
it's just the pandas default:
![image](https://user-images.githubusercontent.com/9868530/146156025-0e709f85-633b-43ca-a4fb-cd0abd50eacd.png)